pkg/maps/ctmap/ctmap_privileged_test.go

```diff
@@ -33,6 +40,25 @@ func (k *CTMapPrivilegedTestSuite) SetUpSuite(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func Benchmark_MapBatchLookup(b *testing.B) {
+	m := newMap(MapNameTCP4Global+"_test", mapTypeIPv4TCPGlobal)
+	err := m.OpenOrCreate()
+	defer m.Map.Unpin()
+	assert.NoError(b, err)
+
+	_ = populateFakeDataCTMap4(m, option.CTMapEntriesGlobalTCPDefault)
+
+	b.ReportAllocs()
+	b.StopTimer()
+	b.ResetTimer()
```

Comment:
nit: I _think_ you don't need the Stop / Start calls; ResetTimer() should be sufficient.